splinterd/src/node/builder/mod.rs

```diff
@@ -185,6 +188,11 @@ impl NodeBuilder {
         self
     }
 
+    pub fn with_auth_configs(mut self, auth_configs: Vec<AuthConfig>) -> Self {
```

Comment:
This is awkward here for two reasons: 1) it exposes actix web 1 code into the testing APIs, potentially causing a lot of test rewriting when we remove actix web 1 support in 0.7 in a couple months; 2) it is decoupled from the variant type; this is only valid for actix web 1, but the API allows setting actix web 3 variant and an actix web 1 auth config.